gluoncv/model_zoo/mobilenet.py

```diff
@@ -43,17 +54,23 @@ def hybrid_forward(self, F, x):
 
 # pylint: disable= too-many-arguments
 def _add_conv(out, channels=1, kernel=1, stride=1, pad=0,
-              num_group=1, active=True, relu6=False):
+              num_group=1, active=True, relu6=False, num_sync_bn_devices=-1):
     out.add(nn.Conv2D(channels, kernel, stride, pad, groups=num_group, use_bias=False))
-    out.add(nn.BatchNorm(scale=True))
+    
+    if num_sync_bn_devices <= 1:
+        out.add(nn.BatchNorm(scale=True))
+    else:
+        out.add(gluon.contrib.nn.SyncBatchNorm(
+            epsilon=1e-5, momentum=0.9, num_devices=num_sync_bn_devices))
```

Comment:
please match the default arguments with the non sync version, nn.BatchNorm(scale=True) and SyncBatchNorm(scale=True, num_devices=num_sync_bn_devices)

Reply:
Okay, I have fixed it.